### ISSUE TITLE
pip-installed tensorflow 2.5.0 cannot use the GPU

### ISSUE THREAD

**Issue by bhack** [googlecolab/colabtools#2013]
**Describe the current behavior**
GPU doesn't work with TF 2.5.0 (any visible device)

**Describe the expected behavior**
GPU runtime work with TF 2.5.0

**What web browser you are using**
Any

**Additional context**
```!pip install tensorflow==2.5.0```

```
May 14, 2021, 4:54:43 PM | WARNING | Skipping registering GPU devices...
-- | -- | --
May 14, 2021, 4:54:43 PM | WARNING | 2021-05-14  14:54:43.022432: W  tensorflow/core/common_runtime/gpu/gpu_device.cc:1766] Cannot dlopen  some GPU libraries. Please make sure the missing libraries mentioned  above are installed properly if you would like to use GPU. Follow the  guide at https://www.tensorflow.org/install/gpu for how to download and  setup the required libraries for your platform.
May 14, 2021, 4:54:43 PM | WARNING | 2021-05-14  14:54:43.022397: I  tensorflow/stream_executor/platform/default/dso_loader.cc:53]  Successfully opened dynamic library libcudnn.so.8
May 14, 2021, 4:54:43 PM | WARNING | 2021-05-14  14:54:43.022052: I  tensorflow/stream_executor/platform/default/dso_loader.cc:53]  Successfully opened dynamic library libcusparse.so.11
May 14, 2021, 4:54:43 PM | WARNING | 2021-05-14  14:54:43.021067: W  tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not  load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11:  cannot open shared object file: No such file or directory;  LD_LIBRARY_PATH: /usr/lib64-nvidia

```

**Comment by colaboratory-team** [comment]
TF 2.5 wheels published to pypi require CUDA 11.2 but colab VMs have only CUDA 11.0.
This will be resolved when we upgrade to 2.5 by default; until then it is expected that `!pip install`ing TF 2.5 will fail to use the GPU.
Tracked internally as b/188060593.

**Comment by colaboratory-team** [comment]
New colab VMs are updated to TF 2.5.0: 
![2 5 0-gpu](https://user-images.githubusercontent.com/38081706/119543021-0ccbad80-bd45-11eb-835b-7bd3840aa80d.png)

If you're still seeing 2.4.1 get a new VM with: `Runtime -> Factory reset runtime`.
